release_notes/AMLS-4702.txt

```diff
@@ -0,0 +1 @@
+ + [AMLS-4702](https://jira.tools.tax.service.gov.uk/browse/AMLS-4702) - 'Converted connectors to use dependency injection'
```

Comment:
Does this need release notes for services DI changes aswell as connectors as they've been merged together?

Reply:
that's release note for my changes only and here I don't need the one for services, as services need to be merged first to master with it's own release note
I've merged services here just to be sure that my changes are working together with changes from branch 4703